content/docs/Tutorials/goFSH/_index.md

```diff
@@ -50,4 +50,4 @@ and place that file into the **fsh/** output directory that was created by GoFSH
 {{< terminal >}} sushi .
 ```
 
-This command will run SUSHI on **resources.fsh**, and generate the output into a directory called **build/**. You can then compare the output from running GoFSH and then SUSHI to the original **StructureDefinition-mcode-genetic-specimen.json** and **StructureDefinition-mcode-laterality.json** files.
```

Comment:
I'm a little confused by what the expected structure of the output will be here, but I think the output location we describe needs to be updated a bit here. If we use SUSHI 1.0.0, GoFSH will be using a "legacy" tank configuration, but I think the output will go to `./input/*` not `build/input/*` (or at least it did when I was testing it). I'm not sure if it's best to update this to say "... and generate the output into a directory called **input/**". Alternatively, if we have a task to update the GoFSH output to go to `input/fsh`, SUSHI will output to the `fsh-generated` directory and this line can just say "... generate the output into a directory called **fsh-generated/**".

Reply:
I thought that output still went to `build/input/*`, and I just tested that and *think* I confirmed it, but you also tested it...so I'm confused.

Reply:
I noticed in the instructions above it says to `cd` into the `fsh` directory.  That's probably what accounts for the difference.  If you run `sushi .` inside of the `fsh` folder, then it treats it like a flat tank -- and writes to `build/input/*`.  If you run it from the parent of `fsh` it treats it like a legacy IG Pub Integration and does not use a `build` folder.  To confuse matters further, these do _not_ produce the same result:
```sh
MyProject/fsh$ sushi . // detected as flat tank mode
MyProject/fsh$ cd ..
MyProject$ sushi ./fsh // detected as legacy IG Pub mode
```
Fun, eh?
```

Reply:
We should try to get GoFSH to produce the new structure soon... but I don't know if it's appropriate to do it before the official 1.0.0 release (as opposed to release candidate).  Hmmmm...

Reply:
Ohh I must have glossed over the "navigate to the fsh/ directory" part. Oops. So I think the `build/input/` part is right now, but I get one profile and one extension, which go into separate folders within `build/input/`. Do you think this should just refer to `build/input/` and leave off the subdirectories? Or should I just be fine with this and wait until we update GoFSH at some point?